packages/mobile/capacitor/capacitorApi.ts

```diff
@@ -102,19 +102,31 @@ export const CapacitorApi: IPlatform = {
         return
     },
 
-    exportTransactionHistory: async (defaultPath, content) => new Promise<string>((resolve, reject) => {}),
+    exportTransactionHistory: async (fileName, content) => {
```

Comment:
Function signatures of `exportTransactionHistory` and `exportMigrationLog` should match. i.e. same order, same parameter names

Reply:
I changed that 👍  But had to modify some other files to not break desktop